### PR TITLE
Move the remaining test dependencies under src/tests

### PR DESCRIPTION
**Pull request by trylek**
This change continues my previous CoreCLR test cleanups by depleting most of the
src/coreclr/tests/src/Common folder (except for the PerfHarness project I'm deleting
in a separate PR). For now I have suppressed building the F# test dependency project
(src/tests/Common/test_dependency_fs/test_dependency.fsproj) as it doesn't build
and no tests actually require its presence.

Thanks

Tomas

### CODE REVIEW

File: src/coreclr/tests/src/dirs.proj [hunk 1]
```diff
@@ -7,19 +7,14 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <DisabledProjects Include="$(TestRoot)TestWrappers*\**\*.csproj" />
       <DisabledProjects Include="$(TestRoot)*\**\cs_template.csproj" />
-      <DisabledProjects Include="$(TestRoot)Common\Coreclr.TestWrapper\Coreclr.TestWrapper.csproj" />
```
Comment (jashook) [inline src/coreclr/tests/src/dirs.proj:12]: What happened to these projects? How are they disabled with the change now?

**Review by jashook — APPROVED** [review]

**Comment by am11** [comment]
Outerloop compiler errors are reported here #42125 (unrelated to this change).

File: src/coreclr/tests/src/dirs.proj (hunk 1 above)
Comment (trylek) [reply src/coreclr/tests/src/dirs.proj:12]: I am adding a global blocking clause for everything under Common in this change, I believe that to be far easier than manually disabling each test. Frankly speaking I'm not super happy about the Common name, I was thinking about renaming it to something like "eng", but I think it's a bit of busywork that can easily wait.

**Comment by trylek** [comment]
I believe the change to be correct even though the inability to run outerloop validation is somewhat frustrating. @jkoritzinsky is hoping to get that fixed by EOD today; if it turns out to be more involved, we can temporarily bug the unmanaged test in issues.targets to unblock the pipeline.

**Review by safern — COMMENTED** [review]
LGTM

**Pull request merged**